homeassistant/components/backup/http.py

```diff
@@ -42,12 +42,13 @@ async def get(
 
         manager = cast(BackupManager, request.app[KEY_HASS].data[DATA_MANAGER])
         backup = await manager.async_get_backup(slug=slug)
+        path = await manager.async_get_backup_path(slug=slug)
```

Comment:
I'd expect us to download the backup from the agent here. If the agent already has it downloaded it doesn't need to do it again of course. The downloaded backup will always have a local path on disk.